test/parsing.js

```diff
@@ -97,7 +97,7 @@ describe('MWV parser; sentence with true wind data', function() {
                             }
                           },
                           "angleTrueWater": {
-                            "value": -42,
+                            "value": -0.7330382858376184,
```

Comment:
Yes. Check also https://github.com/SignalK/signalk-parser-nmea0183/issues/36 - getting the conversions updated to the proper SI units is priority #1, but fixing precision at the same time would not hurt.